sigh/lib/assets/resign.sh

```diff
@@ -81,8 +84,7 @@ warning() {
 function checkStatus {
 
     # shellcheck disable=SC2181
-    if [ $? -ne 0 ];
-    then
+    if [ $? -ne 0 ]; then
```

Comment:
Specific reason for these unrelated syntax changes?

Reply:
Yes, it was hard to know which style to use so I changed them to match what seemed to be the style that is most in use for this file. That's the second commit so it is easy to review the changes.